### DISCUSSION TITLE
Japan's "White Day", held annually on March 15, is for what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#389] [Galaxy Trivia] [answered]
**Hint:** 🎁

**Comment by peterbaccaro** [comment]
**White Day** is actually held exactly one month after Valentines Day on the **14th March** and its purpose if for **men to give gifts to the women who gave them gifts on Valentine's Day**.

**Comment by thuongtruong1009** [comment, marked as answer]
Japan's "White Day," held annually on March 15th, is a holiday where men who received gifts from women on Valentine's Day, which is celebrated on February 14th, return the favor by giving gifts to women. The tradition started in the 1980s, when the confectionery industry in Japan began promoting Valentine's Day as an occasion for women to give chocolates to men as a token of affection. White Day was created as a way for men to reciprocate the gesture and show their appreciation for the women who gave them gifts on Valentine's Day. The name "White Day" comes from the idea that the gifts should be white or marshmallow-themed, such as white chocolate or marshmallow candies.
